python/src/scipp/compat/mantid.py

```diff
@@ -189,8 +192,15 @@ def load(filename="",
     :rtype: Dataset
     """
 
-    import mantid.simpleapi as mantid
-    from mantid.api import EventType
+    try:
+        import mantid.simpleapi as mantid
+        from mantid.api import EventType
+    except ImportError as e:
+        raise ImportError(
```

Comment:
I'm not sure if we would say it here, or in the sphinx documentation, but using docker also a solution.

Reply:
I don't want to suggest that. Using Docker isn't a good solution to this and everyone that currently wants to use this will have access to a Linux system (unless I'm mistaken on that).